src/plugins/paginate/paginate.js

```diff
@@ -115,16 +118,47 @@ const componentName = "wb-paginate",
 			paginationUI = "<ol class=\"pagination\">";
 
 			// Add Previous page button
-			paginationUI += "<li" + ( i === currPage ? " class=\"disabled\"" : "" ) + "><a class=\"paginate-prev\" aria-controls=\"" + elm.id + "\" href=\"#" + elm.id + "\">" + i18nText.prv + "</a></li>";
+			var prevLI = "";
+			prevLI += "<li" + ( i === currPage ? " class=\"disabled\"" : "" ) + ">";
+
+			if ( buttonsUI ) {
+				prevLI += "<button class=\"paginate-prev\" aria-controls=\"" + elm.id + "\"><span class=\"wb-inv\">Page </span>" + i18nText.prv + "</button>";
```

Comment:
Just realized the buttons need a `type="button"` as the browser defaults all buttons to be submit buttons.
```suggestion
				prevLI += "<button type=\"button\" class=\"paginate-prev\" aria-controls=\"" + elm.id + "\"><span class=\"wb-inv\">Page </span>" + i18nText.prv + "</button>";
```